src/webots/nodes/WbPropeller.cpp

```diff
@@ -230,6 +236,9 @@ void WbPropeller::prePhysicsStep(double ms) {
     dBodyAddForceAtPos(b, thrustVector.x(), thrustVector.y(), thrustVector.z(), cot.x(), cot.y(), cot.z());
     dBodyAddTorque(b, torqueVector.x(), torqueVector.y(), torqueVector.z());
 
+    mCurrentThrust = thrust;
+    mCurrentTorque = torque;
+
```

Comment:
I believe the `thrust` and `torque` variables are not needed any more and we can directly compute the value in the ` mCurrentThrust` and `mCurrentTorque` member fields.